server/sonar-server/src/main/java/org/sonar/server/es/EsUtils.java

```diff
@@ -116,10 +123,11 @@ public static String escapeSpecialRegexChars(String str) {
 
     private final Queue<SearchHit> hits = new ArrayDeque<>();
 
-    private DocScrollIterator(EsClient esClient, String scrollId, Function<Map<String, Object>, D> docConverter) {
+    private DocScrollIterator(EsClient esClient, SearchResponse scrollResponse, Function<Map<String, Object>, D> docConverter) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "DocScrollIterator" constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)